src/pudl/extract/eia176.py

```diff
@@ -1,26 +1,39 @@
-"""Extract EIA Form 176 data from CSVs.
+"""Extract EIA Form 176 data from CSVs."""
 
-The EIA Form 176 archive also contains CSVs for EIA Form 191 and EIA Form 757.
-"""
+from dagster import Output, asset
 
-from dagster import asset
+from pudl.extract.csv import CsvExtractor
+from pudl.extract.extractor import GenericMetadata, raw_df_factory
 
-from pudl.extract.csv import CsvExtractor, get_table_file_map
 
-DATASET = "eia176"
+class Extractor(CsvExtractor):
+    """Extractor for EIA form 176."""
 
+    def __init__(self, *args, **kwargs):
+        """Initialize the module.
+
+        Args:
+            ds (:class:datastore.Datastore): Initialized datastore.
+        """
+        self.METADATA = GenericMetadata("eia176")
+        super().__init__(*args, **kwargs)
+
+    def get_page_cols(self, page: str, partition_key: str) -> list[str]:
+        """Get the columns for a particular page and partition key."""
+        # The page columns for EIA176 do not vary by year
+        return super().get_page_cols(page, "any_year")
```

Comment:
I thought maybe I'd be able to bypass the storing of column mappings in a separate CSV like we do for Excel extractors, but this lined up better with how we're using factories and just ends up being more consistent and flexible overall.

Reply:
Do you mind just spelling out in the docstring what you're doing here for future reference - e.g., "EIA 176 data has the same set of columns for all years, so regardless of the partition key provided we select the same columns here" or something like this?